src/Nox.Generator/Common/GeneratorConfig.cs

```diff
@@ -9,5 +12,13 @@ internal class GeneratorConfig
         public bool Infrastructure { get; set; } = true;
 
         public bool Presentation { get; set; } = true;
+
+        public UiType Ui { get; set; } = UiType.None;
```

Comment:
@anna-naumova25  we need to update this for a bool value, sorry we can catch up on this today